templates/definition/meter/sonnenbatterie_eco56.yaml

```diff
@@ -48,5 +51,37 @@ render: |
     source: http
     uri: http://{{ .host }}:{{ .port }}/rest/devices/battery
     jq: .M30 # SOC relative to usable capacity (.M05 # display SOC)
+  batterymode:
+    source: switch
+    switch:
+      - case: 1 # normal
+        set:
+          source: http
+          method: PUT
+          uri: http://{{ .host }}:{{ .port }}/rest/devices/battery/C06
+          body: '10' # Automatic
+      - case: 2 # hold
+        set:
+          source: http
+          method: PUT
+          uri: http://{{ .host }}:{{ .port }}/rest/devices/battery/C06
+          body: '20' # Standby
+      - case: 3 # charge
+        set:
+          source: sequence
+          set:
+            - source: http
+              method: PUT
+              uri: http://{{ .host }}:{{ .port }}/rest/devices/battery/C06
+              body: '55' # slave mode
+            - source: sleep
+              duration: 1s
+            - source: watchdog
```

Comment:
Where are you resetting the watchdog? 30s vs 3m mismatches.

Reply:
The 3 minutes mentioned in the documentation is a maximum that if hit will pause the charging. Meaning that we have to set the value in shorter intervals like `2m59s`.
But the API is not stable. Sometimes there are short periods (usually 1-2m) where the battery just returns 500s. Thus I choose 30 seconds so that we retry four times before the charging is stopped.

Reply:
Der Watchdog hier läuft einfach immer weiter, auch wenn auch normal gestellt wird. Der muss an andere Stelle (siehe andere Templates) und braucht ein reset.

Reply:
Die anderen templates hatte ich mir angeschaut, eigentlich brauche ich einen watchdog aber nur für den Ladefall. Nur in dem Modus hört der Speicher mit dem Laden auf, wenn keine kontinuerlichen requests geschickt werden.
So wie ich die aktuelle konfiguration verstehe würde das bedeuten, dass bei mode `1` und `2` ein reset ausgelöst werden muss was aktuell nicht möglich ist.

Reply:
Ich habe den fehlenden support für mehrere reset werte in c34d8df6b85582a3f11ee180972bbcfe4b5a2f99 eingebaut.